client/css/style.css

```diff
@@ -1345,10 +1345,30 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	margin-top: 10px;
 }
 
-#loading-slow {
+#loading-slow,
+#loading-reload {
 	display: none;
 }
 
+#loading-reload {
+	margin-top: 15px;
+}
+
+#loading summary {
+	outline: none;
+	cursor: pointer;
+}
+
+#loading pre {
+	white-space: normal;
+
+	-webkit-user-select: text;
+	-moz-user-select: text;
+	-ms-user-select: text;
+	user-select: text;
+	cursor: text;
```

Comment:
@xPaw, I'm guessing you made the `pre` content-editable so people could copy-paste. I have instead made it selectable, let me know if you see something wrong in that.

Reply:
Instead of duplicating these rules, can you add the selector where these are defined instead?

Reply:
Done.